cedar-policy-core/src/ast/entity.rs

```diff
@@ -203,7 +324,16 @@ impl EntityUID {
 
 impl std::fmt::Display for EntityUID {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "{}::\"{}\"", self.entity_type(), self.eid.escaped())
+        match self {
+            EntityUID::EntityUID(entity_uid) => write!(
+                f,
+                "{}::\"{}\"",
+                self.entity_type(),
+                entity_uid.eid.escaped()
+            ),
+            #[cfg(feature = "tolerant-ast")]
+            EntityUID::Error => write!(f, "{}::\"ExprError\"", self.entity_type()),
```

Comment:
You don't this specialization, right? We could just use the `entity_uid.eid()` instead of `ExprError`.

Reply:
Woops, good point